tt_metal/impl/dispatch/program_command_sequence.hpp

```diff
@@ -22,7 +22,9 @@ class CircularBuffer;
 
 struct ProgramCommandSequence {
     HostMemDeviceCommand preamble_command_sequence;
-    HostMemDeviceCommand stall_command_sequence;
+    HostMemDeviceCommand* stall_command_sequence_ptr;
```

Comment:
please initialize it with nullptr. I'd probably create a class with private section to hide logic behind this pointer.

Reply:
Based on the suggestion further below, this has been modified to the following:

```
constexpr uint32_t UncachedStallSequenceIdx = 0;
constexpr uint32_t CachedStallSequenceIdx = 1;

uint32_t current_stall_seq_idx = 0;
HostMemDeviceCommand stall_command_sequences[2];
 ```
 
 We don't use raw ptrs for this functionality anymore